CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.10 (2018-05-09)
+-- Bump paper-handlebars to 4.0.1 [#141](https://github.com/bigcommerce/paper/pull/141) to fix 
```

Comment:
This is not true -- we don't use paper-handlebars in the 2.x branch

Reply:
@mattolson fixed